### PR TITLE
feat(hss): support quotas datasource

### PR DESCRIPTION
**Pull request by jinyangyang222**
**What this PR does / why we need it**:
Support quotas datasource
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
Support quotas datasource

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
$ make testacc TEST="./huaweicloud/services/acceptance/hss" TESTARGS="-run TestAccDataSourceQuotas_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/hss -v -run TestAccDataSourceQuotas_basic -timeout 360m -parallel 4
=== RUN   TestAccDataSourceQuotas_basic
=== PAUSE TestAccDataSourceQuotas_basic
=== CONT  TestAccDataSourceQuotas_basic
--- PASS: TestAccDataSourceQuotas_basic (73.01s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/hss       73.061s

```

### CODE REVIEW

File: huaweicloud/services/hss/data_source_huaweicloud_hss_quotas.go [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+	for _, v := range quotas {
+		rst = append(rst, map[string]interface{}{
+			"id":                      v.ResourceId,
+			"version":                 v.Version,
+			"status":                  v.QuotaStatus,
+			"used_status":             v.UsedStatus,
+			"host_id":                 v.HostId,
+			"host_name":               v.HostName,
+			"charging_mode":           convertChargingMode(v.ChargingMode),
+			"expire_time":             utils.FormatTimeStampRFC3339(*(v.ExpireTime)/1000, false),
```
Comment (deer-hang) [inline huaweicloud/services/hss/data_source_huaweicloud_hss_quotas.go]: ditto

File: huaweicloud/services/hss/data_source_huaweicloud_hss_quotas.go [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+}
+
+func flattenTags(tags *[]hssv5model.TagInfo) map[string]interface{} {
+	if tags == nil {
+		return nil
+	}
+
+	rst := make(map[string]interface{})
+	for _, tag := range *tags {
+		rst[*tag.Key] = tag.Value
```
Comment (deer-hang) [inline huaweicloud/services/hss/data_source_huaweicloud_hss_quotas.go]: ditto

File: huaweicloud/services/hss/data_source_huaweicloud_hss_quotas.go (hunk 1 above)
Comment (jinyangyang222) [reply huaweicloud/services/hss/data_source_huaweicloud_hss_quotas.go]: done

File: huaweicloud/services/hss/data_source_huaweicloud_hss_quotas.go (hunk 2 above)
Comment (jinyangyang222) [reply huaweicloud/services/hss/data_source_huaweicloud_hss_quotas.go]: done.

**Pull request merged**